server.rb

```diff
@@ -0,0 +1,28 @@
+require 'sinatra'
+require 'multi_json'
+require 'net/http'
+require 'uri'
+
+configure do
+  set :port, 5000
+end
+
+get '/' do
+  send_file 'index.html'
+end
```

Comment:
Good catch. Corrected this and also corrected server.py which was a little bit of an anti-example serving all content in the app root as static content.